spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/index.adoc

```diff
@@ -33,6 +33,10 @@ include::getting-started.adoc[]
 
 include::{dataflow-asciidoc}/streams.adoc[]
 
+include::{dataflow-asciidoc}/tasks.adoc[]
```

Comment:
Should this be guarded by omit-tasks-docs `ifdef`s (just in case we want to switch back), or are we dropping that 'feature' altogether?